app/backend/controllers/auth.js

```diff
@@ -0,0 +1,8 @@
+module.exports.isAuthenticated = (req, res, next) => {
```

Comment:
Clean, simple and useful. Thanks for your effort, it will come in handy in the future.